NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--<add key="opcua-preview" value="https://opcfoundation.pkgs.visualstudio.com/opcua-netstandard/_packaging/opcua-preview/nuget/v3/index.json" />-->
```

Comment:
Shouldn't this file be removed since everything is commented out in it ?

Reply:
Quite handy if you need to test preview releases

Reply:
OK. I guess it is quite fine for `2.5` as we only do maintenance on it.